Makefile

```diff
@@ -19,9 +19,11 @@ HELM_CLIENT_VER_MAJ := $(shell echo ${HELM_CLIENT_VER} | awk -F. '{print $$2}')
 GIT_HEAD := $(shell git rev-list -1 HEAD)
 GIT_LAST_TAG := $(shell git describe --abbrev=0 --tags)
 GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+GIT_PATCH := "1"
```

Comment:
This GIT_PATCH should be "" on master branch, and we can update the patch in the stable branch

Reply:
GIT_PATCH is updated to "" on master branch.

Reply:
> This GIT_PATCH should be "" on master branch, and we can update the patch in the stable branch

to update patch in stable branch, should new jira be created?